### PR TITLE
docs(ddx): ddx console requirements update

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Dedibox console requirements updated.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: dedibox-console/classic-hosting/how-to/access-logcenter.mdx [hunk 1]
```diff
@@ -17,10 +17,10 @@ categories:
...
 The logcenter allows you to visualize real-time error logs of your hosting. That way, you can quickly troubleshoot potential issues linked to your code.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have created a **Webhosting Classic**
-</Message>
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/access-logcenter.mdx:23]:
```suggestion
- Created a **Webhosting Classic** plan
```

File: dedibox-console/classic-hosting/how-to/access-statistics.mdx [hunk 2]
```diff
@@ -17,10 +17,10 @@ categories:
...
 With each of our Webhosting Classic offers, we provide the possibility to visualize the statistics of your website (number of visits, number of unique visitors, bandwidth used, etc…).
 
-<Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have created a **Webhosting Classic**
-</Message>
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/access-statistics.mdx:23]:
```suggestion
- Created a **Webhosting Classic** plan
```

File: dedibox-console/classic-hosting/how-to/change-email-account-password.mdx [hunk 3]
```diff
@@ -12,13 +12,13 @@ dates:
...
 <Macro id="webhosting-online" />
 
+<Macro id="requirements" />
 
-<Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have created a **Webhosting Classic**
-</Message>
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/change-email-account-password.mdx:21]:
```suggestion
- Created a **Webhosting Classic** plan
```

File: dedibox-console/classic-hosting/how-to/change-ftp-account-password.mdx [hunk 4]
```diff
@@ -15,10 +15,10 @@ categories:
...
 <Macro id="webhosting-online" />
 
-<Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have created a **Webhosting Classic**
-</Message>
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/change-ftp-account-password.mdx:21]:
```suggestion
- Created a **Webhosting Classic** plan
```

File: dedibox-console/classic-hosting/how-to/change-password-mysql-database.mdx [hunk 5]
```diff
@@ -15,11 +15,11 @@ categories:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have created a **Webhosting Classic**
-  - You have [created](/dedibox-console/classic-hosting/how-to/create-mysql-database) a MySQL database
-</Message>
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/change-password-mysql-database.mdx:21]:
```suggestion
- Created a **Webhosting Classic** plan
```

File: dedibox-console/classic-hosting/how-to/change-password-postgresql-database.mdx [hunk 6]
```diff
@@ -15,19 +15,19 @@ categories:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have created a **Webhosting Classic**
-  - You have [created](/dedibox-console/classic-hosting/how-to/create-postgresql-database) a PostgreSQL database
-</Message>
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/change-password-postgresql-database.mdx:21]:
```suggestion
- Created a **Webhosting Classic** plan
```

File: dedibox-console/classic-hosting/how-to/check-emails.mdx [hunk 7]
```diff
@@ -15,11 +15,11 @@ categories:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net)
-  - You have created a **Webhosting Classic**
-  - You have configured an email account
-</Message>
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/check-emails.mdx:21]:
```suggestion
- Created a **Webhosting Classic** plan
```

File: dedibox-console/classic-hosting/how-to/configure-backup-option.mdx [hunk 8]
```diff
@@ -30,6 +27,11 @@ Because you can never be safe against human failure, a hardware fault, or piracy
   * More information about this option and its pricing is available on our [website](https://www.scaleway.com/en/dedibox/cloud-hosting/#sauvegarde).
 </Message>
 
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- Created a **Webhosting Classic**
```
Comment (RoRoJ) [inline dedibox-console/classic-hosting/how-to/configure-backup-option.mdx:33]:
```suggestion
- Created a **Webhosting Classic** plan
```

**Review by RoRoJ — COMMENTED** [review]
I started adding "plan" everywhere because to me it's really weird to say "a Webhosting Classic", but then I don't know if it's worth it since this is a legacy product...whatever you think.

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**